linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/bash/shared.sh

```diff
@@ -2,7 +2,7 @@
 . /usr/share/scap-security-guide/remediation_functions
 
 if ! `rpm -q --quiet chrony` && ! `rpm -q --quiet ntp-`; then
```

Comment:
When you are at it, I would apply the DeMorgan rule on the condition, and those backticks are redundant.